blocks/fragment/fragment.js

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
I believe this piece of code is just a copy/paste from block collection ?

Reply:
yes

Reply:
ok, please make sure fragment issue doesn't get closed since css changes are pending. Good to merge this to unblock.

Reply:
@ujjwal5 , for styling the columns block in fragment, I have created another issue, it would be tracked there.
Closing this issue is not a problem. 
Hope you saw my note in the description above